### PR TITLE
Various improvements to accessibility and behaviour

### PR DESCRIPTION
**Pull request by erbridge**
See the commits for details.

Screenshot 2023-01-28 at 15 26 27
Screenshot 2023-01-28 at 15 25 54
Screenshot 2023-01-28 at 16 02 05

### CODE REVIEW

File: components/messages.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function Messages({ events, isProcessing }) {
     if (events.length > 2) {
       messagesEndRef.current.scrollIntoView({ behavior: "smooth" });
     }
-  });
+  }, [events.length]);
```
Comment (zeke) [inline components/messages.js:14]: Thank you for this! Was gonna fix it but forgot. ⚡

File: components/messages.js [hunk 2]
```diff
@@ -1,54 +1,78 @@
...
+                      <UndoIcon className="icon" /> Undo and try a different
+                      change
+                    </button>
+                  </div>
+                )}
+              </Message>
+
+              {(isProcessing || index < events.length - 1) && (
+                <Message sender="replicate" isSameSender>
+                  What should we change?
```
Comment (zeke) [inline components/messages.js:53]: It seems like this only needs to be shown to the user once to get them going, but I'm seeing a second time after generating an image:

Screenshot 2023-01-29 at 6 41 07 PM

File: components/footer.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+  Download as DownloadIcon,
+  Info as InfoIcon,
+  XCircle as StartOverIcon,
+} from "lucide-react";
 import Link from "next/link";
 
 export default function Footer({ events, startOver, handleImageDropped }) {
   return (
-    <div className="w-full">
+    <footer className="w-full">
```
Comment (zeke) [inline components/footer.js:12]: The footer could use some breathing room up top while the first prediction is running.

 
Screenshot 2023-01-29 at 6 43 15 PM

**Review by zeke — APPROVED** [review]
This is great! I was secretly hoping you would come along and make these kinds of changes. :heart:

I left a few suggestions, but nothing blocking. Feel free to ship this. It'll auto-deploy.

File: components/messages.js (hunk 2 above)
Comment (erbridge) [reply components/messages.js:53]: So it's serving the purpose of the label for the input field. Inputs should have visible labels. We could show a different message, though.

**Pull request merged**